src/dal_select2/static/autocomplete_light/select2.js

```diff
@@ -48,6 +49,43 @@
                 cache: true
             },
         });
+
+        $(this).on('select2:selecting', function (e) {
+            var data = e.params.args.data;
+
+            if (data.create_id !== true)
+                return;
+
+            e.preventDefault();
+
+            var select = $(this);
+
+            $.ajax({
+                url: $(this).attr('data-autocomplete-light-url'),
+                type: 'POST',
+                dataType: 'json',
+                data: {
+                    text: data.id,
+                },
+                beforeSend: function(xhr, settings) {
+                    xhr.setRequestHeader("X-CSRFToken", document.csrftoken);
+                },
+                success: function(data, textStatus, jqXHR ) {
+                    var option = '<option value="' + data.id + '">' + data.text + '</option>'
+                    select.append(option);
+                    select.select2('close');
+
+                    if (select.attr('multiple') == 'multiple') {
```

Comment:
Instead of checking the attribute, it's recommended to check the underlying DOM property with `.prop('multiple')`. This also has the advantage of returning a boolean instead of (hopefully) `'multiple'`.